src/components/Home/PhotoCarousel.jsx

```diff
@@ -12,6 +12,8 @@ import image5 from '../../images/carousel/image5.jpg';
 import image6 from '../../images/carousel/image6.jpg';
 import image7 from '../../images/carousel/image7.jpg';
 
+import { LazyLoadImage } from 'react-lazy-load-image-component';
```

Comment:
nit: Let's move this to be around line 6, still maintaining a space between it and the images.

I think it'd just for readability long term, we can keep the packages at the beginning and for this particular component since (currently) there's a bunch of images we have to manually import, those can be separated.